attitude_error_mon/att_err_mon.py

```diff
@@ -0,0 +1,354 @@
+#!/usr/bin/env python
+
+import os
+import argparse
+import jinja2
+import numpy as np
+import matplotlib
+
+if __name__ == "__main__":
+    matplotlib.use("Agg")
+import matplotlib.pyplot as plt
+
+from astropy.table import Table, vstack, Column
+from astropy.io import ascii
+from Ska.Matplotlib import plot_cxctime
+from Chandra.Time import DateTime
+from kadi import events
+from Ska.engarchive import fetch
+from cxotime import CxoTime
+
+
+FILE_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+def get_options():
+    parser = argparse.ArgumentParser(description="Update attitude error plots")
+    parser.add_argument("--outdir", default=".")
+    parser.add_argument("--datadir", default=".")
+    parser.add_argument(
+        "--recent-start",
+        help="Date to use as reference for highlighted/recent data,"
+        + " default is to highlight last 60 days",
+    )
+
+    opt = parser.parse_args()
+    return opt
```

Comment:
I suggest removing the `parse_args` statement. That way, one can generate documentation automatically from the parser. Even if you don't use it now, I think it's a good pattern.

```suggestion
    return parser
```